Development/nmos/streamcompatibility_resources.cpp

```diff
@@ -0,0 +1,195 @@
+#include "nmos/streamcompatibility_resources.h"
+
+#include <unordered_set>
+#include "nmos/capabilities.h" // for nmos::fields::constraint_sets
+#include "nmos/is11_versions.h"
+#include "nmos/resource.h"
+#include "nmos/streamcompatibility_state.h"
+
+namespace nmos
+{
+    namespace experimental
+    {
+        web::json::value make_streamcompatibility_active_constraints_endpoint(const web::json::value& constraint_sets, bool locked)
+        {
+            using web::json::value_of;
+
+            auto active_constraint_sets = value_of({
+                { nmos::fields::constraint_sets, constraint_sets },
+            });
+
+            return value_of({
+                { nmos::fields::active_constraint_sets, active_constraint_sets },
+                { nmos::fields::temporarily_locked, locked },
```

Comment:
I think the codebase generally avoids trailing comma as some compilers still warn about it.

```suggestion
                { nmos::fields::temporarily_locked, locked }
```

Reply:
Fixed.